packages/pn-pa-webapp/src/models/Institutions.ts

```diff
@@ -0,0 +1,53 @@
+export interface Institution {
```

Comment:
Check if all exported interface are used outside this file. if not remove the export

Reply:
Duplicate